config/param-validation.js

```diff
@@ -5,7 +5,9 @@ module.exports = {
   createUser: {
     body: {
       username: Joi.string().required(),
-      mobileNumber: Joi.string().regex(/^[1-9][0-9]{9}$/).required()
+      // eslint-disable-next-line max-len,no-useless-escape
+      email: Joi.string().regex(/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/).required(),
```

Comment:
Joi actually has a built in email function so we don't need this huge regex
```suggestion
      email: Joi.string().email({ minDomainSegments: 2 }).required(),
```
and then we won't need the `eslint-disable` comment either


Aside: here's some fun reading on email regexes: https://www.regular-expressions.info/email.html